runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -92,7 +92,9 @@ public void onDecorativeObjectSpawned(DecorativeObjectSpawned event)
 		DecorativeObject object = event.getDecorativeObject();
 
 		if (event.getDecorativeObject().getId() == ObjectID.BLUE_TEARS ||
-			event.getDecorativeObject().getId() == ObjectID.BLUE_TEARS_6665)
+			event.getDecorativeObject().getId() == ObjectID.BLUE_TEARS_6665 ||

```

Comment:
At this point, it might be worth storing these in a list, and using `TEARS_OBJECTS.contains(event.getDecorativeObject().getId())` instead.

Reply:
Set, not a list